components/builder-web/app/package-versions-page/_package-versions-page.scss

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2016-2017 Chef Software Inc. and/or applicable contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+.hab-pkg-version {
+  &--left {
+    @include span-columns(8);
+
+    .pkg-container {
+      @include span-columns(8 of 8);
+      padding: rem(10);
+      border-bottom: 1px solid $light-gray;
+    }
+
+    .pkg-col-1 {
+      text-align: left;
+      @include span-columns(4 of 8);
+      font-family: $heading-font-family;
+      font-size: rem(12);
+      text-transform: uppercase;
+    }
+
+    .pkg-col-2 {
+      text-align: right;
```

Comment:
There's another `text-align` defined on 39, so we should probably remove this one.